_extras/Recipe.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: "Online Workshop Recipe"
+---
+
+# A Recipe for an Online Carpentries Workshop
+
+In the past year+ of teaching online, our community has learned many things. Our initial [online training recommendations](https://carpentries.org/online-workshop-recommendations/#planning-your-workshop) recognized that we were in uncharted territory for Carpentries workshops! In consulting with community members, we heard many different narratives and suggestions. Rather than creating a single 'recipe,' we chose to distill that advice into 'things to think about', giving Instructors space to engineer their own solutions to the challenges presented by teaching online.
+
+We continue to be excited to learn from the innovations of our Instructor community! However, at this point we have also developed a better idea about the specific advantages and disadvantages of different practices. For those who do not wish to 'reinvent the wheel' but prefer to start with a recommended model, we offer an example structure for an online workshop that we expect to function well.
```

Comment:
We can embed this link https://docs.carpentries.org/topic_folders/hosts_instructors/index.html#resources-for-online-workshops where the CDT shared online resource from the community